### PR TITLE
Add termynal

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds termynal to the docs site. This should be used moving forward as a consistent and easy-to-maintain alternative to taking screenshots of the terminal.

Goes with mkdocs PR: https://github.com/papermoonio/moonbeam-mkdocs/pull/157

I updated the Moonbeam dev node page as an example of how to use termynal and what the end result looks like 👀 
⭕️ After initial review, I'll create a PR for the CN repo that uses the new code snippets for the terminal output.

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### Corresponding PRs

Please link to any corresponding PRs here.

### CODE REVIEW

File: .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/compile.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <br> Compiling cumulus-client-network v0.1.0 (https://github.com/paritytech/cumulus?branch=ro
+    coco-v1#9d89ed65)
+    <br> Compiling cumulus-client-consensus-relay-chain v0.1.0 (https://github.com/paritytech/cum
+    ulus?branch=rococo-v1#9d89ed65)
+    <br> Compiling polkadot-test-service v0.8.29 (https://github.com/paritytech/polkadot?branch=r
+    ococo-v1#b64741e6)
+    <br> Compiling cumulus-client-collator v0.1.0 (https://github.com/paritytech/cumulus?branch=r
+    ococo-v1#9d89ed65)
+    <br> Compiling cumulus-client-service v0.1.0 (https://github.com/paritytech/cumulus?branch=ro
+    coco-v1#9d89ed65)
```
Comment (themacexpert) [inline .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/compile.md:20]: Can you remove the new lines here between ro and coco? 
Screenshot 2024-01-10 at 8 10 10 PM

**Review by themacexpert — APPROVED** [review]
Very sleek!! I'm a huge fan of this instead of wrangling with screenshots!

**Review by albertov19 — APPROVED** [review]
LGTM!

File: .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/compile.md (hunk 1 above)
Comment (eshaben) [reply .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/compile.md:20]: good catch! thank you!

**Pull request merged**